WORKING-GROUP-PROCESSES.md

```diff
@@ -19,8 +19,7 @@ Istio working groups are organizations responsible for the design and implementa
 Working groups operate with a fair amount of autonomy within the broader scope of the project. They tend to be long-lived, representing major
 initiatives over Istio’s lifetime.
 
-Some working groups focus on specific technologies. For example, the Policy and Telemetry working group primarily focuses on Mixer. Other
-working groups are cross-cutting in nature such as the Performance and Scalability working group.
+Some working groups focus on specific technologies. For example, the Policy and Telemetry working group primarily focuses on Mixer.
```

Comment:
This is not a great example

Reply:
Agreed. Replaced it with one that might be better.